### PR TITLE
[IOTDB-4910] Fix the concurrency problem of GenerateQueryWorkLoad

### PR DESCRIPTION
**Pull request by SpriCoder**
See: https://issues.apache.org/jira/browse/IOTDB-4910

### CODE REVIEW

File: core/src/main/java/cn/edu/tsinghua/iotdb/benchmark/tsdb/DBWrapper.java [hunk 1]
```diff
@@ -696,17 +696,17 @@ private void doErrorLog(String queryName, Status status1, Status status2) {
...
+      //        double timeInMillis = status.getTimeCost() / NANO_TO_MILLIS;
+      //        String formatTimeInMillis = String.format("%.2f", timeInMillis);
+      //        String currentThread = Thread.currentThread().getName();
+      //        LOGGER.info(
+      //            "{} complete {} with latency ,{}, ms ,{}, result points",
+      //            currentThread,
+      //            operation,
+      //            formatTimeInMillis,
+      //            status.getQueryResultPointNum());
+      //      }
```
Comment (JackieTien97) [inline core/src/main/java/cn/edu/tsinghua/iotdb/benchmark/tsdb/DBWrapper.java:709]: Don't need these any more?

File: configuration/conf/config.properties [hunk 2]
```diff
@@ -84,7 +84,7 @@ VERIFICATION_STEP_SIZE=10000
 
 ############## 数据：设备、传感器、客户端 ##################
 # 设备总数
-DEVICE_NUMBER=5
+DEVICE_NUMBER=2
```
Comment (JackieTien97) [inline configuration/conf/config.properties:87]: change it back to previous default value?

File: configuration/conf/config.properties [hunk 3]
```diff
@@ -96,7 +96,7 @@ TS_ALIGNMENT_RATIO=1.0
 # 是否将设备绑定给客户端，如果绑定，则客户端数小于等于设备数，否则可以大于
 IS_CLIENT_BIND=true
 # 客户端总数
-CLIENT_NUMBER=5
+CLIENT_NUMBER=2
```
Comment (JackieTien97) [inline configuration/conf/config.properties:99]: same as above

File: configuration/conf/config.properties [hunk 4]
```diff
@@ -212,7 +212,7 @@ DATA_SEED=666
 # Q8  最近点查询 select time, v1... where device = ? and time = max(time)
 # Q9  倒序范围查询（只限制起止时间）select v1... from data where time > ? and time < ? and device in ? order by time desc
 # Q10 倒序带值过滤的范围查询 select v1... from data where time > ? and time < ? and v1 > ? and device in ? order by time desc
-OPERATION_PROPORTION=1:0:0:0:0:0:0:0:0:0:0
+OPERATION_PROPORTION=0:0:1:0:0:0:0:0:0:0:0
```
Comment (JackieTien97) [inline configuration/conf/config.properties:215]: same as above

File: configuration/conf/config.properties [hunk 5]
```diff
@@ -319,7 +319,7 @@ CSV_OUTPUT=true
 
 ################# 输出结果：日志参数 ######################
 # 是否使用静默模式，静默模式会关闭部分日志输出
-IS_QUIET_MODE=true
+IS_QUIET_MODE=false
```
Comment (JackieTien97) [inline configuration/conf/config.properties:322]: same as above

**Review by JackieTien97 — CHANGES_REQUESTED** [review]

File: core/src/main/java/cn/edu/tsinghua/iotdb/benchmark/tsdb/DBWrapper.java (hunk 1 above)
Comment (SpriCoder) [reply core/src/main/java/cn/edu/tsinghua/iotdb/benchmark/tsdb/DBWrapper.java:709]: （Sorry, forget to move back

File: configuration/conf/config.properties (hunk 5 above)
Comment (SpriCoder) [reply configuration/conf/config.properties:322]: Get.

File: configuration/conf/config.properties (hunk 4 above)
Comment (SpriCoder) [reply configuration/conf/config.properties:215]: Get.

File: configuration/conf/config.properties (hunk 3 above)
Comment (SpriCoder) [reply configuration/conf/config.properties:99]: Get.

**Comment by SpriCoder** [comment]
@JackieTien97  PTAL~

**Pull request merged**